### PR TITLE
fix: strict container check

### PR DESCRIPTION
**Pull request by vvo**

### CODE REVIEW

File: lib/widget-utils.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
-var isString = require('lodash/lang/isString');
-
-function getContainerNode(value) {
-  if (isString(value)) {
-    return document.querySelector(value);
+function getContainerNode(selectorOrHTMLElement) {
+  if (typeof selectorOrHTMLElement === 'string') {
+    return document.querySelector(selectorOrHTMLElement);
+  } else if (!isDomElement(selectorOrHTMLElement)) {
```
Comment (pixelastic) [inline lib/widget-utils.js]: You can remove the `else` has you `return`ed in the previous `if`

Comment (vvo) [inline lib/widget-utils.js]: Not sure to follow, there are three return cases here

Comment (bobylito) [inline lib/widget-utils.js]:
```js
if () {
  return
}
if (){
  return
}
```
Is what @pixelastic meant :)

Comment (pixelastic) [inline lib/widget-utils.js]: Just saying that
```javascript
if (typeof selectorOrHTMLElement === 'string') {
    return document.querySelector(selectorOrHTMLElement);
} else if (!isDomElement(selectorOrHTMLElement)) {
    throw new Error('Container must be `string` or `HTMLElement`');
}
```
is equal to
``` javascript
if (typeof selectorOrHTMLElement === 'string') {
    return document.querySelector(selectorOrHTMLElement);
}
if (!isDomElement(selectorOrHTMLElement)) {
    throw new Error('Container must be `string` or `HTMLElement`');
}
```

(the `else` is useless)

Comment (vvo) [inline lib/widget-utils.js]: allright, done

**Pull request merged**